prov/efa/test/efa_unit_test_cq.c

```diff
@@ -894,16 +897,18 @@ void test_efa_cq_read_send_success(struct efa_resource **state)
 {
 	struct efa_resource *resource = *state;
 	struct efa_unit_test_buff send_buff;
+	struct fi_context2 ctx;
 	struct fi_cq_data_entry cq_entry;
 	fi_addr_t addr;
 	int ret;
 
-	test_efa_cq_read(resource, &addr, IBV_WC_SEND, IBV_WC_SUCCESS, 0);
+	test_efa_cq_read(resource, &addr, IBV_WC_SEND, IBV_WC_SUCCESS, 0,
+			 (struct efa_context *) &ctx);
 	efa_unit_test_buff_construct(&send_buff, resource, 4096 /* buff_size */);
 
 	assert_int_equal(g_ibv_submitted_wr_id_cnt, 0);
 	ret = fi_send(resource->ep, send_buff.buff, send_buff.size,
-		      fi_mr_desc(send_buff.mr), addr, (void *) 12345);
+		      fi_mr_desc(send_buff.mr), addr, &ctx);
 	assert_int_equal(ret, 0);
 	assert_int_equal(g_ibv_submitted_wr_id_cnt, 1);
 
```

Comment:
Not related to this PR, but generally it worth checking the completion flags in the cq entry to be correct.